core/src/test/java/tachyon/master/LocalTachyonClusterMultiMaster.java

```diff
@@ -205,9 +200,8 @@ public void start() throws IOException {
     }
 
     mWorker =
-        TachyonWorker.createWorker(
-            CommonUtils.parseInetSocketAddress(mCuratorServer.getConnectString()),
-            new InetSocketAddress(mLocalhostName, 0), 0, 1, 100, mWorkerConf);
+        TachyonWorker.createWorker(mCuratorServer.getConnectString().split(":")[0],
```

Comment:
@hsaputra  Or we can remove the TachyonWorker.createWorker(String masterAddress, int masterPort, int dataPort, int minWorkerThreads, int maxWorkerThreads, TachyonConf tachyonConf) API, just keep the TachyonWorker.createWorker(TachyonConf) API as the only way to create a worker. What is your suggestion?